Cargo.toml

```diff
@@ -8,7 +8,6 @@ members = [
     "lib/optics",
     "lib/prelude",
     "lib/shapely/impl",
-    "lib/shapely/macros",
```

Comment:
Oh, I see the test script is sometimes problematic. On my branch I've created something like a "black list" for crates in workspace, so soon we can do this in somewhat less hacky way